Simulator/python/src/Estimator.py

```diff
@@ -67,26 +112,82 @@
     }
 }
 
+default_basic_options = {
+    'r': {
+        'description': 'number of rows of pixels in frame',
+        'type': 'number',
+        'value': 1080
+    },
+    'c': {
+        'description': 'number of columns of pixels in frame',
+        'type': 'number',
+        'value': 1920
+    }
+}
+
 EstimatorDocs = {
+    "basic_linear": {
+        'name': 'basic_linear',
+        'description': 'Basic linear interpolation',
+        'options': default_basic_options
+    },
+    "basic_frame_rep": {
+        'name': 'basic_frame_rep',
+        'description': 'Basic frame repetition interpolation',
+        'options': default_basic_options
+    },
+    "basic_oversampling": {
+        'name': 'basic_oversampling',
+        'description': 'Basic oversampling interpolation',
+        'options': default_basic_options
+    },
+
     "MEMCI_unidir_fs": {
         'name': 'MEMCI_unidir_fs',
         'description': 'MEMCI, Unidirectional, Full Search',
         'options': default_fs_options
     },
+    "MEMCI_unidir_tss": {
+        'name': 'MEMCI_unidir_tss',
+        'description': 'MEMCI, Unidirectional, Three Step Search',
+        'options': default_tss_options
+    },
     "MEMCI_unidir_hbma": {
         'name': 'MEMCI_unidir_hbma',
         'description': 'MEMCI, Unidirectional, Hierarchical Block Matching Algorithm',
         'options': default_hbma_options
     },
+
     "MEMCI_bidir_fs": {
         'name': 'MEMCI_bidir_fs',
         'description': 'MEMCI, Bidirectional, Full Search',
         'options': default_fs_options
     },
+    "MEMCI_bidir_tss": {
+        'name': 'MEMCI_bidir_tss',
+        'description': 'MEMCI, Bidirectional, Three Step Search',
+        'options': default_tss_options
+    },
     "MEMCI_bidir_hbma": {
         'name': 'MEMCI_bidir_hbma',
         'description': 'MEMCI, Bidirectional, Hierarchical Block Matching Algorithm',
         'options': default_hbma_options
+    },
+
+    "MEMCI_bidir2_fs": {
+        'name': 'MEMCI_bidir2_fs',
+        'description': 'MEMCI, Bidirectional 2, Full Search',
+        'options': default_fs_options
+    },
+    "MEMCI_bidir2_tss": {
+        'name': 'MEMCI_bidir2_tss',
+        'description': 'MEMCI, Bidirectional 2, Three Step Search',
+        'options': default_tss_options
+    },
+    "MEMCI_bidir2_hbma": {
+        'name': 'MEMCI_bidir2_hbma',
+        'description': 'MEMCI, Bidirectional 2, Hierarchical Block Matching Algorithm',
+        'options': default_hbma_options
```

Comment:
Has these been tested from CLI + gui?

Reply:
have*

Reply:
Cli works but I have no idea what's going on with the gui I haven't looked

Reply:
I think @NGovani  is trying?

Reply:
Yeah will check today, just need to build the gui on our branch